chord_wes/runner.py

```diff
@@ -28,22 +28,63 @@
 INGEST_POST_TIMEOUT = 60 * 10  # 10 minutes
 
 
+def ingest_in_drs(path):
+    # TODO: might want to refactor at some point
+    url = f"http+unix://{NGINX_INTERNAL_SOCKET}/api/drs/ingest"
+    params = {"path": path}
+
+    try:
+        r = requests.post(url, json=params, timeout=INGEST_POST_TIMEOUT)
+        r.raise_for_status()
+    except requests.RequestException:
+        return None
+
+    data = r.json()
+
+    for access_method in data["access_methods"]:
+        if access_method["type"] == "file":
+            return access_method["access_url"]["url"]
```

Comment:
This returns the file path, not the `drs://` URL - shouldn't it return the latter?

Reply:
Indeed you are right, I just sent a fix